src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IForLoopStatement.cs

```diff
@@ -1686,14 +1686,14 @@ public class F
                                 Left: 
                                   IFieldReferenceOperation: System.Int32 F.i (OperationKind.FieldReference, Type: System.Int32) (Syntax: 'i')
                                     Instance Receiver: 
-                                      IInstanceReferenceOperation (OperationKind.InstanceReference, Type: F, IsImplicit) (Syntax: 'i')
+                                      IInstanceReferenceOperation (ReferenceKind: ImplicitReceiver) (OperationKind.InstanceReference, Type: F, IsImplicit) (Syntax: 'i')
```

Comment:
Thanks, this is very useful information. Do we have an issue to track exposing a helper extension method that maps an `IInstanceReferenceOperation` with an `ImplicitReceiver` to the receiver operation? That will be a useful helper for dataflow analysis visitor, where when you encounter an `IInstanceReferenceOperation`, you normally want to return the tracked state/data for the receiver being referenced.

Reply:
Not currently, can you file one?

Reply:
Filed https://github.com/dotnet/roslyn/issues/26152